### DISCUSSION TITLE
How can I pass HeaderRowStyling.Class

### DISCUSSION THREAD

**Discussion by SandroRiz** [Megabit/Blazorise#3011] [Q&A] [answered]
I had an old project with 
<DataGrid ...
                  HeaderRowClass="thead-dark"

that is now removed and afaik (reading code) substituted by HeaderRowStyling.Class
sorry for my ignorance, but how can I pass inside the markup ?

<DataGrid ...
                   HeaderRowStyling.Class = "thead-dark"

doesn't work.

Thanks

**Comment by stsrki** [comment, marked as answer]
Use

`HeaderRowStyling="@(new DataGridRowStyling { Class = "something"; })"`

**Reply by SandroRiz** [reply]
Thanks it works (but without ";")
